frontend/src/components/pages/message/MessageView.tsx

```diff
@@ -2,99 +2,60 @@
 // SPDX-FileCopyrightText: 2010-2021 Dirk Riehle <dirk@riehle.org
 // SPDX-FileCopyrightText: 2019 Georg Schwarz <georg. schwarz@fau.de>
 
-import React, { useState } from 'react'
-import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
-import ExpandLessIcon from '@mui/icons-material/ExpandLess'
-import { Collapse, CardActions, Button } from '@mui/material'
+import React from 'react'
 
 const MessageView: React.FC<MessageViewProps> = ({ data }) => {
-	const [expanded, setExpanded] = useState(false)
+	const {
+		messageId,
+		topic,
+		payload,
+		schema,
+		namespace,
+		tenant,
+		publishTime,
+		producer,
+	} = data
 
-	const handleExpand = () => {
-		//TODO if(!data) fetch detailed data
-		setExpanded(!expanded)
-	}
+	const publishDate = new Date(publishTime)
 
 	return (
 		<div className="flex flex-col card-content">
-			<h2 className="uppercase">{data?.payload}</h2>
-			<div className="grey-line"></div>
+			<h2 className="uppercase">{messageId}</h2>
```

Comment:
Maybe it would be great to add a title for messageId like `Message ID: {messageId}`